integrations/ragas/tests/test_evaluator.py

```diff
@@ -293,6 +283,7 @@ def test_evaluator_outputs(current_metric, inputs, expected_outputs, metric_para
 # This integration test validates the evaluator by running it against the
 # OpenAI API. It is parameterized by the metric, the inputs to the evaluator
 # and the metric parameters.
+@pytest.mark.asyncio
```

Comment:
(A bit unrelated)

Ragas recently made several changes in how they execute evaluations. (https://github.com/explodinggradients/ragas/pull/1093)

I encountered several failures like this: https://github.com/deepset-ai/haystack-core-integrations/actions/runs/10057773128/job/27799695446

Adding the `pytest-asyncio` plugin and this marker makes tests run correctly.